### ISSUE TITLE
SOLVED: Google Domains DNS, failed authorization procedure

### ISSUE THREAD

**Issue by jcward** [Fmstrat/easywildcardnew#7]
Hi, I followed the instructions carefully, creating DNS entries for my domain like this in the Google Domains DNS settings:

![image](https://user-images.githubusercontent.com/2192439/92829592-9f801400-f391-11ea-8a36-200a66a685ba.png)

And when I run the script, I get:
```
Failed authorization procedure. example.com (dns-01): urn:ietf:params:acme:error:dns :: DNS problem: NXDOMAIN looking up TXT for _acme-challenge.example.com - check that a DNS record exists for this domain, example.com (dns-01): urn:ietf:params:acme:error:dns :: DNS problem: NXDOMAIN looking up TXT for _acme-challenge.example.com - check that a DNS record exists for this domain
```

**Solution:** I've worked with multiple DNS services, and I could swear Google is the first one that's let me make the mistake of entering the domain name in what's supposed to be strictly the prefix field on the left.

In other words, all 3 records need to look like this (**_NO_** domain name in the left column, yes domain in the right column):

![image](https://user-images.githubusercontent.com/2192439/92829877-f259cb80-f391-11ea-808a-404655da1158.png)

Hopefully my mistake saves someone else some trouble. :+1:
